plugins/plugin-svelte/plugin.js

```diff
@@ -17,23 +17,25 @@ module.exports = function plugin(snowpackConfig, {hot: hotOptions, ...sveltePlug
     svelteRollupPlugin({include: '**/node_modules/**', dev: isDev}),
   );
 
-  let svelteOptions;
+  let {config = '.', ...svelteOptions} = sveltePluginOptions || {};
+  let userSvelteOptions;
   let preprocessOptions;
-  // Note(drew): __config is for internal testing use; maybe we should make this public at some point?
-  const userSvelteConfigLoc =
-    sveltePluginOptions.__config || path.join(process.cwd(), 'svelte.config.js');
+
+  const userSvelteConfigLoc = path.resolve(process.cwd(), `${config}/svelte.config.js`);
```

Comment:
configuring the directory but not the file is always confusing to me as a user, so lets go all the way and configure the file: 

```
// config is always absolute, or relative to the cwd
let {config = 'svelte.config.js', ...svelteOptions} = sveltePluginOptions || {};
const userSvelteConfigLoc = config || path.resolve(process.cwd(), `svelte.config.js`);
```

Reply:
I'm fine with verbose clarity. I'll update to `configFilePath` and update that path + filename needs to be configured.

Reply:
Great! much appreciated